projects/adrv9001/a10soc/system_qsys.tcl

```diff
@@ -2,7 +2,7 @@ source $ad_hdl_dir/projects/scripts/adi_pd.tcl
 source $ad_hdl_dir/projects/common/a10soc/a10soc_system_qsys.tcl
 source ../common/adrv9001_qsys.tcl
 
-set_instance_parameter_value sys_spi {clockPolarity} {1}
+set_instance_parameter_value sys_spi {clockPolarity} {0}
```

Comment:
I would have removed this line since clockPolarity is anyway set to 0 in a10soc_system_qsys.tcl (https://github.com/SRaus/hdl/blob/master/projects/common/a10soc/a10soc_system_qsys.tcl#L271). It wouldn't make any difference from the functional point of view, but keeping this line might be confusing.